### DISCUSSION TITLE
Unable to signin to github

### DISCUSSION THREAD

**Discussion by HaiderAli532** [community/community#83922] [Codespaces]
### Select Topic Area

Bug

### Body

![Screenshot 2024-01-02 173859](https://github.com/community/community/assets/153230833/1198b9f9-f144-490d-a5f7-e9670fff0d4a)
I am unable to signin to github
By clicking on "Sign in to Github" button, nothing happens

**Comment by Shahnawazk7944** [comment]
**I think you only want to sign in to GitHub not GitHub codespaces**
**here are the steps to sign in to GitHub in VSCode**

**1. Install Git:** Download and install Git ([https://git-scm.com/download/win](https://git-scm.com/download/win)).

**2. Install Extension:** In VS Code, search and install "GitHub Pull Requests and Issues" extension.

**3. Sign in to GitHub:** Click the GitHub icon in VS Code, then "Sign in" and follow prompts.

**4. Clone/Publish:**
    - **Clone existing:** Click "Clone Repository", paste URL, choose local folder.
    - **Create new:** Click "Publish to GitHub", follow prompts to create repo and clone.

**5. Commit/Push:**
    - Edit code, stage changes in Source Control panel.
    - Enter commit message, click "Commit".
    - Click "Sync Changes" to push to GitHub.

**Tips:** Use Command Palette (Ctrl+Shift+P) for quick Git commands.

Now you're ready to rock with GitHub in VS Code!
